opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectAttackPatternsKillChainLines.tsx

```diff
@@ -204,11 +201,7 @@ const StixDomainObjectAttackPatternsKillChainLines: FunctionComponent<StixDomain
                                     )}
                                   </IconButton>
                                 ) : (
-                                  <StixCoreRelationshipPopover
-                                    stixCoreRelationshipId={attackPattern.id}
-                                    paginationOptions={paginationOptions}
-                                    onDelete={onDelete}
-                                  />
+                                  <></>
```

Comment:
This is OK, but I think it would be cleaner if this was simplified to
```
{ coursesOfAction ? (
   component
)}
```
now that we no longer do anything when coursesOfAction=false